kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -0,0 +1,112 @@
+// @flow
+import * as React from 'react';
+import SubHeader from '../components/subheaders/index.jsx';
+
+export const title = 'MDN Web Docs payment terms';
+const TermsPage = () => (
+    <>
+        <SubHeader title={title} />
+        <main
+            className="contributions-page"
+            role="main"
+            data-testid="terms-page"
+        >
+            <section>
+                <h2>Payment Terms</h2>
```

Comment:
It's a bit outside my comfort zone to say but I find it odd that the...

* Document title is: `MDN payment terms`
* The `<h1>` is: `MDN Web Docs payment terms`
* and the `<h2>` is: `Payment Terms`

They are all different styles. I don't know English writing enough to know if it's supposed to be "These Payment Terms cover your payment..." or "These payment terms cover your payment..."
But I think the mishmash phrasing looks untidy and scrappy.

Reply:
Good point. The American vs. European casing is really inconsistent across MDN and it's bothered me! I changed it to "Payment Terms" across the board, and removed the first <h2> since it's redundant if it's already in the page title.